apps/api-documenter/package.json

```diff
@@ -22,7 +22,7 @@
     "js-yaml": "~3.9.1"
   },
   "devDependencies": {
-    "@microsoft/rush-stack-compiler": "0.5.4",
+    "@microsoft/rush-stack-compiler-3.0": "0.0.0",
```

Comment:
> "0.0.0", [](start = 41, length = 9)

0.0.0?

Reply:
It'll get bumped to 0.1.0 when it's published.

---
In reply to: [236908556](https://github.com/Microsoft/web-build-tools/pull/947#discussion_r236908556) [](ancestors = 236908556)